packages/fluentui/react-northstar/src/themes/teams/components/Datepicker/datepickerCalendarGridStyles.ts

```diff
@@ -7,6 +7,6 @@ export const datepickerCalendarGridStyles: ComponentSlotStylesPrepared<
   DatepickerVariables
 > = {
   root: (): ICSSInJSStyle => {
-    return { tableLayout: 'fixed' };
+    return { 'border-spacing': '0rem', tableLayout: 'fixed' };
```

Comment:
If you want to try keep the datepicker the same size as before, you can add padding around each of the datepicker cells, the user agent stylesheet for me on edge applies a 2px border spacing around each cell by default

Reply:
This might be preferrable since you'd have less visual regressions